### DISCUSSION TITLE
Does VirtualC64 emulate REU RAM and SuperCPU expanded RAM

### DISCUSSION THREAD

**Discussion by senf666** [dirkwhoffmann/virtualc64#734] [Q&A]
Does VirtualC64 emulate REU RAM and SuperCPU expanded RAM. I would like to run a program but it requires this ram expansion. Is it possible with  VirtualC64 ? 

"Only REU RAM and SuperCPU expanded RAM are currently supported at this time.  If you don't have any such RAM, there are a few modern cartridges currently being sold that provide an emulated solution (like the 1541 Ultimate II+).  C64 emulators like VICE also support REU emulation"

https://visionbasic.net

**Comment by dirkwhoffmann** [comment]
No, sorry, neither of the two are supported yet.

**Reply by senf666** [reply]
Thanks for the info.
I'll raise a feature request for this.

**Reply by DNSGeek** [reply]
+1
